universal-application-tool-0.0.1/app/services/applicant/ReadOnlyApplicantProgramService.java

```diff
@@ -10,8 +10,8 @@
 
   /**
    * Get the {@link Block}s for this program and applicant. This includes all blocks, whether the
-   * block was filled out in this program or a previous program. This also includes blocks that may
-   * be hidden from the applicant (i.e. they have a show/hide predicate).
+   * block was filled out in this program or a previous program. This will not include blocks that
+   * are hidden from the applicant (i.e. they have a show/hide predicate).
    */
   ImmutableList<Block> getAllBlocks();
```

Comment:
Suggestion (non-blocking): I feel that `getAllBlocks` not technically returning all blocks a bit confusing here. Can we name this something like `getAllActiveBlocks`?

Reply:
I think you're right, I like `getAllActiveBlocks` a lot more. Will update!